### PR TITLE
[wxwidgets] Update to 3.2.5

### LINKED ISSUE #38731: [wxwidgets] update to 3.2.5
### Library name

wxwidgets

### New version number

3.2.5

### Other information that may be useful (release notes, etc...)

[Changelog](https://raw.githubusercontent.com/wxWidgets/wxWidgets/v3.2.5/docs/changes.txt)
[Readme](https://raw.githubusercontent.com/wxWidgets/wxWidgets/v3.2.5/docs/readme.txt)

### PR DESCRIPTION
**Pull request by jimwang118**
Fixes https://github.com/microsoft/vcpkg/issues/38731
Delete upstream merged patch fix-glegl.patch.
- [X] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [X] SHA512s are updated for each updated download.
- [ ] ~~The "supports" clause reflects platforms that may be fixed by this new version.~~
- [ ] ~~Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~~
- [X] Any patches that are no longer applied are deleted from the port's directory.
- [X] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [X] Only one version is added to each modified port's versions file.

All features passed with following triplets:

```
x86-windows
x64-windows
x64-windows-static
```

### CODE REVIEW

File: ports/wxwidgets/portfile.cmake [hunk 1]
```diff
@@ -12,7 +12,7 @@ vcpkg_from_github(
         fix-pcre2.patch
         gtk3-link-libraries.patch
         sdl2.patch
-        fix-glegl.patch
+        #fix-glegl.patch
```
Comment (dg0yt) [inline ports/wxwidgets/portfile.cmake:15]:
```suggestion
```

File: ports/wxwidgets/vcpkg.json [hunk 2]
```diff
@@ -35,10 +34,7 @@
     "nanosvg",
     "opengl",
     "pcre2",
-    {
-      "name": "tiff",
-      "default-features": false
-    },
+    "tiff",
```
Comment (dg0yt) [inline ports/wxwidgets/vcpkg.json:37]: NO!
Why can't I rely on vcpkg maintainers to not add damage to hard work? :angry:

File: ports/wxwidgets/vcpkg.json [hunk 3]
```diff
@@ -35,10 +34,7 @@
     "nanosvg",
     "opengl",
     "pcre2",
```
Comment (dg0yt) [inline ports/wxwidgets/vcpkg.json:36]: This is what you should change!
```suggestion
    {
      "name": "pcre2",
      "default-features": false
    },
```

**Review by dg0yt — CHANGES_REQUESTED** [review]

**Review by WangWeiLin-MV — APPROVED** [review]
The port installation tests pass with the following triplets:
* x64-windows

**Review by data-queue — APPROVED** [review]

**Pull request merged**